sphinx/source/docs/user_guide/compat.rst

```diff
@@ -75,15 +60,25 @@ requires only one line of code in HoloViews:
  :align: center
 
 Adding overlaid plots, slider widgets, selector widgets, selection
-tools, and tabs is similarly straightforward. HoloViews objects can
-also be rendered using a Matplotlib-based backend, which allows SVG or
-PDF output not currently available for native Bokeh plots. See the
-Holoviews `Bokeh_Backend`_ tutorial for more details.
+tools, and tabs is similarly straightforward.
+
+Further synergy
+---------------
+
+You can render HoloViews objects with the `Matplotlib`_ library. This
+enables SVG and PDF output not currently available for native Bokeh
+plots. See the HoloViews `tutorial`_ on Bokeh backend for more
```

Comment:
It seems like there is an article missing in this sentence ("on the Bokeh"). How about:

> For more details, see the `tutorial`_ on the Bokeh backend in the Holoviews documentation.